src/params.js

```diff
@@ -11,6 +11,8 @@ var models = noSlash(env.models || 'ts');
 
 var packageJson = fs.existsSync('./package.json') ? JSON.parse(fs.readFileSync('./package.json')) : {};
 
+const defaultDependencyPath = 'node_modules/-api-dependencies';
```

Comment:
That path looks strange with this leading dash in the 2nd segment. Is this by intent?

Reply:
It was designed this way from the beginning. Maybe it was to avoid clashes with real modules.